Description: Don't throw an error when topcom has a nonzero return value
 Otherwise we get failing tests.
Origin: https://github.com/Macaulay2/M2/commit/25445d9 (revert)
Last-Update: 2020-07-09

--- a/M2/Macaulay2/packages/Topcom.m2
+++ b/M2/Macaulay2/packages/Topcom.m2
@@ -97,7 +97,6 @@
     if debugLevel >= 2 then << "-- " << command << ": executing " << executable << endl;
 
     retval := run executable;
-    if 0 =!= retval then error ("error running topcom:"| net get errfile);
 
     if debugLevel >= 5 then << "-- " << command << ": output = " << net get outfile << endl;
     if debugLevel >= 6 then << "-- " << command << ": stderr = " << net get errfile << endl;    
